jdi-light-angular-tests/src/main/java/io/github/com/pages/sections/TabGroupSection.java

```diff
@@ -0,0 +1,42 @@
+package io.github.com.pages.sections;
+
+import com.epam.jdi.light.angular.elements.complex.ButtonToggle;
+import com.epam.jdi.light.angular.elements.complex.TabGroup;
+import com.epam.jdi.light.ui.html.elements.common.*;
+
+public class TabGroupSection {
+
+    public TabGroup basicTab;
+
```

Comment:
Any blank lines between class fields are redundant.
This may only be necessary to compose field groups.